ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -2325,6 +2325,82 @@ tests:
       WORKFLOW_TYPE: cucushift
       YQ_VERSION: 4.30.5
     workflow: openshift-e2e-ibmcloud-heterogeneous-test-qe
+- as: azure-short-cert-rotation-f9
+  cron: 22 23 2,14,23 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      FEATURE_SET: TechPreviewNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-short-cert-rotation-f9
+  cron: 0 13 1,10,22 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: openshift-e2e-cert-rotation-short
+- as: gcp-short-cert-rotation-f9
+  cron: 22 9 4,16,25 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      FEATURE_GATES: '["ShortCertRotation=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-gcp-ipi
+- as: baremetal-short-cert-rotation-f9
+  cluster: build05
+  cron: 25 15 9,18,27 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ARCHITECTURE: arm64
+      DEVSCRIPTS_CONFIG: |
+        FEATURE_SET=TechPreviewNoUpgrade
+      NVME_DEVICE: /dev/nvme1n1
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: vsphere-short-cert-rotation-f9
```

Comment:
vsphere, ibmcloud, nutanix does not support multi-arch payload yet, so need to keep them in amd64 nightly ci yaml file.